src/renderer/components/common/card/Card.tsx

```diff
@@ -5,12 +5,21 @@ import style from "./Card.module.scss";
 
 interface CardProps {
     title: string;
+    className?: string;
     color?: "blue" | "green" | "grey" | "orange" | "purple";
 }
 
-export const Card: FC<CardProps> = ({ children, title, color = "grey" }) => {
+export const Card: FC<CardProps> = ({
+    children,
+    title,
+    className,
+    color = "grey",
```

Comment:
Magic color (design system)

Reply:
done (design system to implement later tho)